Library/NavigationTests.swift

```diff
@@ -218,6 +218,21 @@ public final class NavigationTests: XCTestCase {
     let url = URL(string: "https://email.kickstarter.com/mpss/c/2gA/Oiw/t.25j/deadbeef/h1/dead-beef")!
     XCTAssertEqual(.emailClick, Navigation.match(url))
 
+    XCTAssertEqual(
+      .emailClick,
+      Navigation.match(URL(string: "https://me.kickstarter.com/wf/click?upn=deadbeef")!)
+    )
+
+    XCTAssertEqual(
+      .emailClick,
+      Navigation.match(URL(string: "https://ea.kickstarter.com/wf/click?upn=deadbeef")!)
+    )
+
+    XCTAssertEqual(
+      .emailClick,
+      Navigation.match(URL(string: "https://clicks.kickstarter.com/wf/click?upn=deadbeef")!)
+    )
+
     XCTAssertEqual(
```

Comment:
Not technically part of this PR, but would be nice to also test `click.e` subdomain.

Reply:
Good eye! Added it 👍